Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/services/impl/AuditServiceImpl.java

```diff
@@ -213,8 +207,9 @@ private String marshallAuditMessage(AuditMessageType mess) {
         if (mess != null) {
             try {
                 JAXBContextHandler oHandler = new JAXBContextHandler();
-                JAXBContext jc = oHandler.getJAXBContext("com.services.nhinc.schema.auditmessage");
+                JAXBContext jc = oHandler.getJAXBContext(AUDIT_MESSAGE_NAMESPACE);
                 Marshaller marshaller = jc.createMarshaller();
+                marshaller.setProperty(Marshaller.JAXB_FORMATTED_OUTPUT, true);
                 ByteArrayOutputStream baOutStrm = new ByteArrayOutputStream();
```

Comment:
If exception happens, the ByteArrayOutputstream will never close it.  Please move it to final block.